.gitignore

```diff
@@ -22,6 +22,7 @@ virtualenv/
 *.rc
 .tox
 ansible/configs/*/roles
+ansible/configs/*/collections
```

Comment:
i believe it is `ansible/collections`

Reply:
for me, it clones them to the config directory :thinking:

Reply:
i created a PR to fix what we did:
https://github.com/redhat-cop/agnosticd/pull/1295

See especially:
https://github.com/redhat-cop/agnosticd/pull/1295/commits/2962a931ea54e9136399746f6591d69a65e8b802

Reply:
Ok, droping :+1: